parachain/runtime/interlay/src/lib.rs

```diff
@@ -896,15 +899,23 @@ impl supply::OnInflation<AccountId> for DealWithRewards {
         // existential deposit is not required due to whitelist
         let _ = Self::Currency::transfer(
             from,
-            &AnnuityAccount::get(),
+            &VaultAnnuityAccount::get(),
             token_distribution::VAULT_INFLATION_REWARDS * amount,
             ExistenceRequirement::KeepAlive,
         );
 
-        // TODO: split stake-to-vote rewards
+        // stake-to-vote rewards
         let _ = Self::Currency::transfer(
             from,
-            &VaultRegistryAccount::get(),
+            &EscrowAnnuityAccount::get(),
+            token_distribution::ESCROW_INFLATION_REWARDS * amount,
+            ExistenceRequirement::KeepAlive,
+        );
+
+        // remainder goes to treasury
+        let _ = Self::Currency::transfer(
+            from,
+            &TreasuryAccount::get(),
             token_distribution::TREASURY_INFLATION_REWARDS * amount,
```

Comment:
To better deal with rounding errors, maybe it makes sense to transfer `amount.saturating_sub(vault_inflation).saturating_sub(escrow_inflation)`?